Tools/autotest/test_build_options.py

```diff
@@ -313,8 +313,18 @@ def check_deps_consistency(self):
         for feature in options:
             self.get_disable_defines(feature, options)
 
+    def check_duplicate_labels(self):
+        '''check that we do not have multiple features with same labels'''
+        options = self.get_build_options_from_ardupilot_tree()
+        seen_labels = {}
+        for feature in options:
+            if seen_labels.get(feature.label, None) is not None:
+                raise ValueError("Duplicate entries found for label '%s'" % feature.label)
+            seen_labels[feature.label] = True
```

Comment:
This could be done a little nicer with a `set` and `feature.label in seen_labels` - but this is fine.